### ISSUE TITLE
Recurring tasks

### ISSUE THREAD

**Issue by idoric** [iman-salmani/iplan#60]
I am well aware that it quickly becomes a real headache¹ if you want to do it well, but do the recurring tasks fall within the scope of this project?

¹ What we call recurring tasks cover several very different things: a task that takes place every 4th of the month, or another every Wednesday and Sunday, or another every two days. And for the latter, if I'm a day late, does that shift all the next rehearsals of the action by one day, or not?

**Comment by iman-salmani** [comment]
Yes, It should be implemented before sync support because most services have this.
I think auto-shifting all next rehearsals to forward can be handy for small periods but also unexpected for the user. Which can be fixed with a popup. (I should see what others have done)

**Comment by idoric** [comment]
Perhaps a source of inspiration: https://gitlab.gnome.org/powimod/aion-task/-/wikis/Aion-Task-Manager-home-page ("Aion-Task is a simple cyclic task manager that allows you to think about all those little things that you have to do frequently and often forget").
